packages/web_drivers/pubspec.yaml

```diff
@@ -8,5 +8,6 @@ dev_dependencies:
   args: 1.5.2
   http: 0.12.0+2
   path: 1.6.4
+  strings: 0.1.2
```

Comment:
I think these version constraints will likely not satisfy the version constraints when we start using this package in `flutter/flutter`. How about switching to version ranges ([example](https://github.com/google/quiver-dart/blob/master/pubspec.yaml))?

Reply:
Also, check out packages used by [flutter_tools](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/pubspec.yaml). The tools do all sorts of path/url/string manipulation, launch processes, establish network connections. I think the list of packages they use should be sufficient for this package too. This way we won't widen Flutter's dependencies, which tend to be sources of flakiness and build reproducibility issues.

Reply:
removed the dep.

Reply:
What about the tight version numbers?

Reply:
I didn't understand the suggestion.

Do you suggest using ranges (like flutter) or do you suggest using strict numbers as flutter_tools (which I am also doing right now)? 
"I think the list of packages they use should be sufficient" -> we are exactly using packages from that list, what is the action item here?

Reply:
The issue is that this package depends on specific versions of packages. If we depend on it from flutter/engine or flutter/flutter `pub` may fail to resolve the version constraints. It will also prevent flutter from upgrading to new package versions. To fix this, simply prepend `^` before every version number, e.g. `^0.1.2`.